### PR TITLE
NPO Profile Page

### PR DESCRIPTION
**Pull request by DanKhoo**
In this PR,
- Added `Profile Panel`, `Review Panel`, `Past Wish Panel`
- Added new `getUserType` api

To Do:
- [x] Past wishes Panel
- [ ] API integration for Reviews [KIV till biz team gets back to us]
- [x] API Integration for checking of user if exists

Libraries Added: 
- `next-images`: Used for importing svg

### CODE REVIEW

**Comment by kohchihao** [comment]
[BUG] if no past wishes, stuck with loading icon.

File: pages/profile/[userId].js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```
Comment (jamessspanggg) [inline pages/profile/[userId].js:1]: Are we using the same endpoint for both donor profile and npo profile? `/profile/[userId].js`? or are we separating them

File: utils/authentication/userType.js [hunk 2]
```diff
@@ -55,3 +55,25 @@ export function isDonor(user, res, destination = {}) {
...
+
+/**
+ * Checks if an user is a Donor
+ * @param {array} userTypes containing the user types
+ */
+export const containsDonor = (userTypes) => {
+  if (userTypes === null) {
+    return false;
+  }
+  return userTypes.type.includes('donor');
```
Comment (jamessspanggg) [inline utils/authentication/userType.js:78]: can use the `userType` constants u created?

File: src/components/imageContainers/ProfileAvatar.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
```
[the remark this answers is not in the document]
Comment (kohchihao) [reply src/components/imageContainers/ProfileAvatar.js:2]: We can. but probably not in this PR.

**Pull request merged**